docs/UserGuide.md

```diff
@@ -76,27 +78,33 @@ Format: `help`
 
 Adds a student to the address book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/SCHEDULE s/SUBJECT f/FEE [paid/PAID] [owed/OWED]`
+**Format:** `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/SCHEDULE s/SUBJECT r/RATE [paid/PAID] [owed/OWED]`
 
-<box type="note" seamless>**NOTE:** 
-1. <b>SCHEDULE</b> must be in the format of `DAY_OF_THE_WEEK`-`START_TIME`-`END_TIME`
-* <b>DAY_OF_THE_WEEK</b> includes Monday, Tuesday, Wednesday, Thursday, Friday, Saturday, Sunday.
-* <b>START_TIME</b> and <b>END_TIME</b> are represented as `HHmm`.
+**Example:**
+* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 t/Sunday-1000-1200 s/Geography r/100 paid/100 owed/0`
 
-2. New clashing schedule will be informed so that you can modify using the [`edit` command](#editing-a-student--edit)
+**Output:**
+![addResult.jpeg](images/addResult.jpeg)
 
-3. <b>SUBJECT</b> should only be:
-`economics` / `literature` / `music` / `biology` / `chemistry` / `science` / `english` / `chinese` / `malay` / `tamil` / `math` / `history` / `geography` / `physics` / `GP`
+<box type="important" header="#### Constraints">
 
-4. <b>FEE</b> is the tuition fee per hour.
+1. **SCHEDULE** must be in the format of `DAY_OF_THE_WEEK`-`START_TIME`-`END_TIME`
+2. **DAY_OF_THE_WEEK** includes `Monday` `Tuesday` `Wednesday` `Thursday` `Friday` `Saturday` `Sunday`
+3. **START_TIME** and <b>END_TIME</b> are represented as `HHmm`.
+4. **PHONE_NUMBER** should be 8 digits that starts with 6, 8 or 9.
+5. **FEE**, <b>PAID</b> and <b>OWED</b> must be at least 0 with at most 2 decimal places.
+      <i>Example: </i> `12.00`, `0.0` or `7`
+6. **SUBJECT** should only be:
+`economics`  `literature`  `music`  `biology`  `chemistry`  `science`  `english`  `chinese`  `malay` `tamil`  `math`  `history`  `geography`  `physics`  `GP`
 
-5. <b>FEE</b>, <b>PAID</b> and <b>OWED</b> must be at least 0 with at most 2 decimal places.
-<i>Example: </i> `12.00`, `0.0` or `7`
 </box>
 
-Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 t/Sunday-1000-1200 s/Geography f/100 paid/100 owed/0`
-* `add n/Betsy Crowe s/Math t/Saturday-1400-1600 e/betsycrowe@example.com a/Block 7 #02-02 Red Hill p/1234567 f/200 owed/200`
+<box type="tip" header="#### Tips">
```

Comment:
I just tried and feel like header 5 would be better HAHA. Header 4 is somehow too big!